flang/lib/Lower/Bridge.cpp

```diff
@@ -106,34 +106,47 @@ using IncrementLoopNestInfo = llvm::SmallVector<IncrementLoopInfo>;
 
 namespace {
 
-/// Walk over the pre-FIR tree (PFT) and lower it to the FIR dialect of MLIR.
-///
-/// After building the PFT, the FirConverter processes that representation
-/// and lowers it to the FIR executable representation.
+/// Create a pre-FIR tree (PFT) from the parse tree, then traverse it to
```

Comment:
I don't think FirConverter actually creates the PFT though. Has that been changed?

It does traverse the PFT and lower to IR.

Reply:
I'll change this to _Traverse the pre-FIR tree (PFT) to generate the FIR dialect of MLIR._